Cannot find module './plejd'

Hi!

Having some issues when starting the addon for the first time..

-----------------------------------------------------------
 Add-on version: 0.1.0
 You are running the latest version of this add-on.
parse error: Expected string key before ':' at line 1, column 4
[23:22:21] ERROR: Unknown HTTP error occured
 System:   (amd64 / qemux86-64)
 Home Assistant version: 0.103.2
 Supervisor version: 193
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[23:22:21] INFO: Starting the Plejd service...
[23:22:21] INFO: Verified permissions on startup script
[23:22:21] INFO: Executing startup script
[23:22:21] INFO: Wrote plejd.json
[23:22:21] INFO: Running add-on
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module './plejd'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/plejd/main.js:1:15)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)

--- a/plejd/main.js
+++ b/plejd/main.js
@@ -1,4 +1,3 @@
-const plejd = require('./plejd');
 const api = require('./api');
 const mqtt = require('./mqtt');
 const fs = require('fs');

